src/main/java/frc/team166/robot/subsystems/Lift.java

```diff
@@ -206,6 +207,7 @@ protected boolean isFinished() {
             protected void end() {
                 liftDrive.stopMotor();
                 engageBrake();
+                liftDrive.set(0);
```

Comment:
This is unnecessary, as the liftDrive.stopMotor() call above will do the same thing. The GoToTopLift command you added needs a end function like this one.